sky/core.py

```diff
@@ -89,6 +89,20 @@ def _start(
             f'Starting cluster {cluster_name!r} with backend {backend.NAME} '
             'is not supported.')
 
+    if cluster_name == spot.SPOT_CONTROLLER_NAME:
+        if down:
+            raise ValueError('Using autodown (rather than autostop) is not '
+                             'supported for the spot controller. Pass '
+                             '`down=False` or omit it instead.')
+        if (idle_minutes_to_autostop is not None and idle_minutes_to_autostop !=
```

Comment:
The second part of the if statement `idle_minutes_to_autostop != spot.SPOT_CONTROLLER_IDLE_MINUTES_TO_AUTOSTOP`, can we just completely disallow the user to specify idle stop time even if it is the right time?

Reply:
Good catch, done.